src/models/product.js

```diff
@@ -1,27 +1,31 @@
-export default (sequelize, DataTypes) => sequelize.define('product', {
-  id: {
-    type: DataTypes.UUID,
-    primaryKey: true,
-    defaultValue: DataTypes.UUIDV4
-  },
-  photoUrl: {
-    type: DataTypes.STRING,
-    allowNull: false
-  },
-  name: {
-    type: DataTypes.STRING,
-    allowNull: false
-  },
-  description: {
-    type: DataTypes.STRING,
-    allowNull: true
-  },
-  price: {
-    type: DataTypes.DECIMAL(10, 2),
-    allowNull: false
-  },
-  marketPrice: {
-    type: DataTypes.DECIMAL(10, 2),
-    allowNull: false
-  }
-})
+export default (sequelize, DataTypes) => { 
+  const Product = sequelize.define('product', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    photoUrl: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    price: {
```

Comment:
With the addition of the group, the price will belong to the group. The market price will still belong to the product.

Reply:
I created the field 'discount' in Group model to deduct the discounted price of the group in the backend. Check it please.